plugin/trino-mongodb/pom.xml

```diff
@@ -232,6 +256,20 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.11.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Please avoid using Mock libraries - For additional details on why (https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-mocks)

Reply:
done